dist/index.js

```diff
@@ -45224,7 +45388,7 @@ function getParameters() {
         const mappingFileInput = core.getInput('mapping-file', { required: false });
         const workspaceFolder = core.getInput('workspace', { required: false });
         const mappingFile = mappingFileInput && (0, app_file_1.validateMappingFile)(mappingFileInput);
-        var env = {};
+        const wait = core.getInput('wait', { required: false }) === 'false' ? false : true;
```

Comment:
Let's rename `wait` to `async` and flip the meaning. The reasoning being that `maestro cloud` command will be blocking by default and to make it non-blocking one would need to specify a `--async` flag (having a `--wait=false` is a bit awkward)

Reply:
Sure, will update!